integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/BuildApplication.java

```diff
@@ -82,8 +82,28 @@
    * @param namespace name of the namespace to create the pod in
    * @return Path path of the archive built
    */
-  public static Path buildApplication(Path appSrcPath, Map<String, String> antParams,
-                                      String antTargets, String archiveDistDir, String namespace) {
+  public static Path buildApplication(
+          Path appSrcPath, 
+          Map<String, String> antParams,
+          String antTargets, String archiveDistDir, String namespace) {
+    return buildApplication(appSrcPath, antParams, antTargets, archiveDistDir, namespace, null);
+  }
+
+  /**
+   * Build application.
+   * @param appSrcPath path of the application source folder
+   * @param antParams ant parameters
+   * @param antTargets ant targets to call
+   * @param archiveDistDir location of the archive built inside source directory
+   * @param namespace name of the namespace to create the pod in
+   * @param targetPath name of the namespace to create the pod in
```

Comment:
Fix this comment. Explain why this targetPath is necessary

Reply:
done
@param targetPath the target path where the application will be archived